packages/atomic/cypress/integration/facets/timeframe-facet/timeframe-facet.cypress.ts

```diff
@@ -407,55 +404,10 @@ describe('Timeframe Facet V1 Test Suites', () => {
 
     describe('verify rendering', () => {
       before(setupTimeframeFacetCustomAmount);
-      CommonFacetAssertions.assertDisplayFacet(TimeframeFacetSelectors, true);
-      CommonAssertions.assertAccessibility(timeframeFacetComponent);
-      CommonFacetAssertions.assertLabelContains(TimeframeFacetSelectors, label);
       CommonAssertions.assertContainsComponentError(
```

Comment:
IMO only test that mattered. The rest (value rendering, select action) is already tested for non-custom timeframes, which are the same. We have no data-set in the future as well, so values might break.

Reply:
Also made relative values bigger so tests won't break in the future, there is always a risk though that if the dataset is not updated, some test fail.